### DISCUSSION TITLE
mogenius 好像要关掉免费套餐了

### DISCUSSION THREAD

**Discussion by asadahimeka** [luolongfei/freenom#208] [General]
![image](https://user-images.githubusercontent.com/31837214/222317220-dab7c7b2-1457-435e-946e-4e63ebaae535.png)
![image](https://user-images.githubusercontent.com/31837214/222317449-e02fc202-509c-49ca-aeaf-be8cb033b4a6.png)

**Comment by luolongfei** [comment]
了解，已更新文档
